eng/pipelines/runtime.yml

```diff
@@ -382,6 +382,25 @@ jobs:
           eq(variables['monoContainsChange'], true),
           eq(variables['isFullMatrix'], true))
 
+# Build and test libraries under single-file publishing
```

Comment:
@safern @ViktorHofer Could one of you take a look at this? I'm trying to add a CI pipeline for single-file but I'm not sure this is right.